templates/magazine/moderators.html.twig

```diff
@@ -49,12 +50,14 @@
                        value="{{ csrf_token('magazine_ownership_request') }}">
             </form>
         {% endif %}
+        {% endif %}
     </div>
-    {% include 'magazine/_moderators_list.html.twig' %}
-    {% if(moderators.haveToPaginate is defined and moderators.haveToPaginate) %}
-        {{ pagerfanta(moderators, null, {'pageParameter':'[p]'}) }}
-    {% endif %}
-    {% if not moderators|length %}
+    {% if moderators|length %}
```

Comment:
found this while testing, list was being included even when no moderators, which is how we display kbin mags now similar to lemmy
before
![image](https://github.com/MbinOrg/mbin/assets/146029455/79bac2d6-ab8f-4e15-96bd-0e26610aaef1)
after
![image](https://github.com/MbinOrg/mbin/assets/146029455/e1719dac-76da-48f3-8847-cdf7d42b153a)